tests/test_extbot_camelcase.py

```diff
@@ -0,0 +1,22 @@
+from telegram.ext import ExtBot
+
+
+def to_camel_case(snake_str):
+    """https://stackoverflow.com/a/19053800"""
+    components = snake_str.split('_')
+    # We capitalize the first letter of each component except the first one
+    # with the 'title' method and join them together.
+    return components[0] + ''.join(x.title() for x in components[1:])
+
+
+class TestCamelCase:
+    def test_camel_case(self):
```

Comment:
please move this into `test_bot.py`. We work with `ExtBot` there anyway. Actually, you can extend the test to run on both `tg.Bot` and `tg.ext.ExtBot` :)

Reply:
Why do this with tg.Bot though?

Reply:
to make sure that all snake_case functions have a `camelCase` equivalent

Reply:
Implemented in https://github.com/python-telegram-bot/python-telegram-bot/pull/2659/commits/922c1e530e441373c34d1ba15d7521b87b60ac81